server/svix-server/src/v1/endpoints/endpoint/mod.rs

```diff
@@ -209,6 +217,7 @@ impl EndpointIn {
 impl ModelIn for EndpointIn {
     type ActiveModel = endpoint::ActiveModel;
 
+    #[allow(deprecated)]
```

Comment:
I don't think I've ever seen this before. Not saying it's wrong, but what's the error message if this isn't there?

Reply:
<img width="595" alt="Screenshot 2023-07-18 at 2 50 27 PM" src="https://github.com/svix/svix-webhooks/assets/133019767/e16e3066-9148-48a4-9948-4ab853289edc">